packages/schema-type/tsconfig.json

```diff
@@ -1,16 +1,8 @@
 {
+  "extends": "../../tsconfig.json",
   "compilerOptions": {
-    /* Visit https://aka.ms/tsconfig.json to read more about this file */
-    "target": "es2020",
-    "module": "esnext",
-    "moduleResolution": "node",
-    "types": [],
-    "declaration": true,
-    "sourceMap": true,
-    "isolatedModules": true,
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "skipLibCheck": true,
-    "strict": true
-  }
+    "outDir": "dist",
+    "types": []
```

Comment:
any reason to explicitly define this as an empty array?

Reply:
It would prevent @types from being resolved, but I'm not sure why I added it here. I'll remove it.